duckduckgo/duckduckgo-privacy-extension
# update icons

**Reviewer:** @MariagraziaAlastra @jagtalon 



## Description:

The badge and search icons are missing a red outline
<img width="500" alt="Screen Shot 2021-03-03 at 2 31 13 PM" src="https://user-images.githubusercontent.com/1882892/109875079-333de900-7c2d-11eb-85ce-8e207eb2885c.png">

## Steps to test this PR:
1. Open about:extensions, or about:addons. 
2. Badge and search icon should have an outline
<img width="333" alt="Screen Shot 2021-03-03 at 2 29 03 PM" src="https://user-images.githubusercontent.com/1882892/109875215-654f4b00-7c2d-11eb-9801-c572186360c8.png">


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
